bin/newrelic-admin

```diff
@@ -71,11 +71,17 @@ USAGE
     # we need to make sure there is a command to run
     print_usage('run-perl') unless @ARGV;
 
-    use NewFangle qw(newrelic_init);
+    use NewFangle qw(newrelic_configure_log newrelic_init);
     require NewFangle::App;
     require NewFangle::Agent;
     require NewFangle::Agent::Config;
 
+    # We use info, warning, error, debug
+    if ( $ENV{NEWRELIC_LOG_LEVEL} ) {
```

Comment:
I think we'll want to read this from the config object, which knows what values to use and in what order they override each other.